### PR TITLE
Malum Psycrosses

### PR DESCRIPTION
**Pull request by lavillastrangiato**
## About The Pull Request

Adds Malum psycrosses. Also slightly edits the Pestran amulet description to be more in line with the other ones.

## Why It's Good For The Game

More faith stuff good. Also means we are one step closer to more Malum templars.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/clothing/rogueclothes/neck.dm [hunk 1]
```diff
@@ -314,10 +314,23 @@
 
 /obj/item/clothing/neck/roguetown/psycross/silver/pestra
 	name = "amulet of Pestra"
-	desc = "A Pestran amulet depicting her fabled walking staff, encrusted in this trinket is a venomous serpent. Believed to ward off disease and lessen poisons to devoted worshippers of the wise hag."
+	desc = "When pure, alcohol is best used as a cleanser of wounds and a cleanser of the palate."
```
Comment (sanshoom) [inline code/modules/clothing/rogueclothes/neck.dm:317]: i’m sorry but this sucks compared to the old one.

Comment (BadAtThisGame302) [reply code/modules/clothing/rogueclothes/neck.dm:317]: The other amulets don't provide a description of the amulet but some sort of quip.... I think that this is why the description is being changed.

Comment (lavillastrangiato) [reply code/modules/clothing/rogueclothes/neck.dm:317]: Sanshoom suggest a quip/doctrine quote I can use :)

**Comment by ThePainkiller** [comment]
# GET KISSED, GET ABSOLUTELY FUCKING KISSED

**Comment by lavillastrangiato** [comment]
> Screenshots of the Malum Amulet?
> 
> Also weird how Malum is the only one to have 2 seperate ones, but whatever.

Teeeechnically Psydon has two different ones? Admittedly, I just included them both because I couldn't decide which one I liked better.
necklace_hammer
necklace_hammer_alt

**Review by NPC1314 — APPROVED** [review]

**Pull request merged**